Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -180,17 +197,24 @@ public class ArcGISARView: UIView {
     /// - Parameter initialTransformation: The initial transformation for originCamera offset.
     /// - Returns: Whether setting the `initialTransformation` succeeded or failed.
     public func setInitialTransformation(initialTransformation: AGSTransformationMatrix) -> Bool {
-        fatalError("setInitialTransformation(initialTransformation:) has not been implemented")
+        _initialTransformation = initialTransformation
+        return true
     }
     
     /// Sets the initial transformation used to offset the originCamera.  The initial transformation is based on an AR point determined via existing plan hit detection from `screenPoint`.  If an AR point cannot be determined, this method will return `false`.
```

Comment:
Should that be "plane"?

```suggestion
    /// Sets the initial transformation used to offset the originCamera.  The initial transformation is based on an AR point determined via existing plane hit detection from `screenPoint`.  If an AR point cannot be determined, this method will return `false`.
```